lib/utils/utils.js

```diff
@@ -12,6 +12,21 @@ module.exports = {
     return !methods || methods.length === 0;
   },
 
+  getAssignmentsToThis: function(node) {
+    return _.filter(node.value.body.body, (item) => {
```

Comment:
I guess you could remove the if statement if you wanted to make it more concise...